include/roaring/containers/bitset.h

```diff
@@ -154,9 +154,9 @@ static inline bool bitset_container_add(bitset_container_t *bitset,
     const int index = pos & 63;
     const uint64_t new_word = old_word | (UINT64_C(1) << index);
     const uint64_t increment = (old_word ^ new_word) >> index;
-    bitset->cardinality += increment;
+    bitset->cardinality += (uint32_t)increment;
     bitset->array[pos >> 6] = new_word;
-    return increment;  // 0 == false, 1 == true
+    return increment > 0;
```

Comment:
warning C4800: 'const uint64_t': forcing value to bool 'true' or 'false' (performance warning)
this is probably easier to read too